### PR TITLE
Adds a thematic quote to HARDDELETES.md

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## Why It's Good For The Game

I'm bored ok tgui is the worst and so are planes.
We stan qt3.14159 goth garbage gargoyles

Also it looks nice and I like it, gets the idea across well

### CODE REVIEW

File: .github/guides/HARDDELETES.md [hunk 1]
```diff
@@ -5,7 +5,17 @@
...
 ## What is Hard Deletion
+
+> Garbage collection is pretty gothic when you think about it. 
+>
+>An object in code is like a ghost, clinging to its former life, and especially to the people it knew. It can only pass on and truly die when it has dealt with its unfinished business. And only when its been forgotten by everyone who ever knew it. If even one other object remembers it, it has a connection to the living world that lets it keep hanging on
+>
+>There is a kind of sombre tone to fixing GC errors too, its almost shamanistic, making sure all these little objects clear up their final affairs in life before they die, to ensure they don't become ghosts
+>
+> -- <cite>Nanako</cite>
+
```
Comment (Kylerace) [inline .github/guides/HARDDELETES.md:18]:
```suggestion

> i like rust and all
>buuut it removes garbage collecctor
>and i pretend garbage collector is a cute girl checking my code
>
> -- <cite>Armhulenn</cite>
```

Comment (Kylerace) [reply .github/guides/HARDDELETES.md:18]: https://tenor.com/view/jolteon-pickle-pokemon-eeveelution-gif-24194135

Comment (tralezab) [reply .github/guides/HARDDELETES.md:18]: alright i agree with it

Comment (LemonInTheDark) [reply .github/guides/HARDDELETES.md:18]: opinion noted and ignored

**Comment by tralezab** [comment]
hey everyone potato force pushes goths

File: .github/guides/HARDDELETES.md [hunk 2]
```diff
@@ -263,3 +276,14 @@ First, do a quick check.
...
 
 If this fails, you're just gonna have to read over this doc. You can skip the theory if you'd like, but it's all pretty important for having an understanding of this problem
+
+## Misc facts
+
+> i like rust and all, buuut it removes garbage collecctor, and i pretend garbage collector is a cute girl checking my code
+>
+> -- <cite>Armhulenn</cite>
+
+- The reference tracker, while powerful, is increadibly easy to break<br>
```
Comment (san7890) [inline .github/guides/HARDDELETES.md:286]:
```suggestion
- The reference tracker, while powerful, is incredibly easy to break<br>
```

Comment (san7890) [reply .github/guides/HARDDELETES.md:286]: I did this on mobile btw

**Comment by Mothblocks** [comment]
i like rust and all, buuut it removes garbage collecctor, and i pretend garbage collector is a cute girl checking my code

File: .github/guides/HARDDELETES.md [hunk 3]
```diff
@@ -263,3 +276,14 @@ First, do a quick check.
 Are you doing anything to the object in `Initialize()` that you don't undo in `Destroy()`? I don't mean like, setting its name, but are you adding it to any lists, stuff like that
 
 If this fails, you're just gonna have to read over this doc. You can skip the theory if you'd like, but it's all pretty important for having an understanding of this problem
+
+## Misc facts
+
+> i like rust and all, buuut it removes garbage collecctor, and i pretend garbage collector is a cute girl checking my code
```
Comment (Mothblocks) [inline .github/guides/HARDDELETES.md:282]: i like rust and all, buuut it removes garbage collecctor, and i pretend garbage collector is a cute girl checking my code

**Pull request merged**